components/ShopList/Foods.tsx

```diff
@@ -39,7 +39,7 @@ const Todos = () => {
     <InfoBox>
       <h1>Food shopping´s list</h1>
       <p>Click an item to cross it out</p>
-      <TextBtn onClick={() => update()}>Activate thunk</TextBtn>
+      <TextBtn onClick={update()}>Activate thunk</TextBtn>
```

Comment:
`update` without invoking the function

Reply:
that doesn´t recognize the function without ()
<img width="723" alt="Captura de Pantalla 2022-07-05 a la(s) 16 43 47" src="https://user-images.githubusercontent.com/81971169/177403656-2d5502ee-9a17-46db-b17a-d121a9fca9a0.png">